tree/ntuple/v7/doc/specifications.md

```diff
@@ -677,6 +677,13 @@ The on-disk representation is similar to a `std::vector<T>` where `T` is the val
   - Child field of type `T`, which must by a type with RNTuple I/O support.
     The name of the child field is `_0`.
 
+### ROOT::Experimental::RNTupleCardinality
+
+A field whose type is `ROOT::Experimental::RNTupleCardinality` should always appear as a "projected" field whose single alias column is the index column of a collection (see Section "Alias columns").
+
+This read-only projected field offers a run-time view over the index column in which each value represents the cardinality of the pointed-to collection.
+The cardinality for the $i$-th element is computed by subtracting the $(i-1)$-th value from the $i$-th value in the index column, e.g. given the index column values `[1, 1, 3]`, the values computed at run-time by the `RNTupleCardinality` field shall be `[1, 0, 2]`.
```

Comment:
I think we need to explain this a bit more: we need to discuss the case where i falls on the start of a cluster (which includes the special case `i == 0`).